bears/general/InvalidLinkBear.py

```diff
@@ -24,8 +25,18 @@ def get_status_code_or_error(url):
         except requests.exceptions.RequestException:
             pass
 
-    @staticmethod
-    def find_links_in_file(file):
+    def get_status_code_checked(self, url):
+        code = InvalidLinkBear.get_status_code(url)
+        if code is None:  # pragma: no cover
+            # if status code of google.com is None, then its
+            # an unstable connection.
+            if self.get_status_code(self.check_connection_url) is None:
```

Comment:
`code is None and self.get_status_code(....`
nested ifs are basically an `and` as it's shortcircuited